src/vos/vos_layout.h

```diff
@@ -169,8 +169,8 @@ struct vos_dtx_act_ent_df {
 	uint64_t			dae_srv_gen;
 	/** The active DTX entry on-disk layout generation. */
```

Comment:
Not in your patch but just noticed the comment is wrong here.   Should be the index identifying the DTX in local container